go-controller/pkg/ovn/gress_policy.go

```diff
@@ -86,11 +87,12 @@ func (gp *gressPolicy) ensurePeerAddressSet(factory AddressSetFactory) error {
 	}
 
 	gp.peerAddressSet = as
+	ipv4HashedAS, ipv6HashedAS := as.GetASHashName()
 	if config.IPv4Mode {
```

Comment:
@JacobTanenbaum should we just do `if ipvrHashedAS != "" {` here since it handles that now in GetASHashName()?

Reply:
yes, I can change that

Reply:
done